astropy/constants/cgs.py

```diff
@@ -3,14 +3,10 @@
 Astronomical and physics constants in cgs units.  See :mod:`astropy.constants`
 for a complete listing of constants defined in Astropy.
 """
-import itertools
-
 from .config import codata, iaudata
 from .constant import Constant
 
-for _nm, _c in itertools.chain(
-    sorted(vars(codata).items()), sorted(vars(iaudata).items())
-):
+for _nm, _c in (*sorted(vars(codata).items()), *sorted(vars(iaudata).items())):
```

Comment:
`sorted` returns a list, so one can just add; it is marginally faster (but, really, this is fine too!)

Reply:
I can only think of only one reason why this line couldn't be written as
```suggestion
for _c in (*vars(codata).values(), *vars(iaudata).values()):
```
That reason is that both `codata` and `iaudata` might contain constants that have the same abbreviation, in which case we want to include only the one for which the corresponding variable name (i.e. `_nm`) comes first alphabetically. It is not at all clear to me that we should allow different constants with the same abbreviation, so this code might have to rewritten much more extensively, but doing that would go far beyond the scope of this pull request.

Reply:
I meant `for _nm, _c in sorted(vars(codata).items()) + sorted(vars(iaudata).items()):` -- but, really, fine to stick with what you have!

Reply:
p.s. And agree that for this PR we should not try to change the meaning of anything, even if "it should work"!